### PR TITLE
ajout du composant Home

### PR DESCRIPTION
**Pull request by gasparddhi**
A valider et a remplacé my profil par Home

### CODE REVIEW

**Review by Erriep — APPROVED** [review]

**Review by Erriep — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
SnE!

**Review by Erriep — DISMISSED** [review]

**Review by Erriep — DISMISSED** [review]

File: src/components/Home/styles.jsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  display: flex;
+  flex-direction: column;
+  
+  li {
+    margin-top: 10px;
+  
+  }
+}
+
+ }
```
Comment (LoicBrassart) [inline src/components/Home/styles.jsx]: Where's my Prettier ?! :x:

File: src/components/Home/styles.jsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  div.containers {
+    border: 2px solid #dbd8d8;
+    box-shadow: 1px 1px #dbd8d8;
+    width: 400px;
+    text-align: center;
+    height: 250px;
+    margin-right: 35px;
+    border-radius: 4%;
+    h3 {
+      font-family: Verdana;
```
Comment (LoicBrassart) [inline src/components/Home/styles.jsx]: For the whole file, I want you to fix your indentations

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Nope!
More style, and more Evils! è_é

**Review by Erriep — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE!

**Pull request merged**